### DISCUSSION TITLE
Partage de ressources pour l'awesome list

### DISCUSSION THREAD

**Discussion by AbcSxyZ** [Open-Models/Brique#5] [General]
Tu peux proposer tes ressources pour l'[awesome list](https://github.com/Open-Models/Brique/blob/main/awesome-list.md) directement en pull requests si tu as quelques compétences avec Git, sinon partage-les dans un commentaire ici pour que quelqu'un puisse ensuite les ajouter.

Toutes les ressources sont bonnes à prendre que ce soit des livres, articles, vidéos, podcasts... Merci d'avance !

**Comment by gibello** [comment]
@AbcSxyZ Lien "awesome list" cassé (404).

**Reply by AbcSxyZ** [reply]
Merci pour le signalement, corrigé. Erreur suite à une migration de dépôt.
